.circleci/config.yml

```diff
@@ -142,12 +142,13 @@ jobs:
         environment:
           JAVA_TOOL_OPTIONS: -Xmx512m # Java can read cgroup. Sadly the cgroup in CircleCI is wrong. Have to manually set. Nothing to do with surefire plugin, it has its own JVM. The two of these must add up to a bit less than 4GB.
           PGHOST: 127.0.0.1
-      - image: circleci/postgres:9.6.19
+      - image: circleci/postgres:11.8
         environment:
           POSTGRES_USER: postgres
           POSTGRES_DB: postgres
           POSTGRES_HOST_AUTH_METHOD: trust
           POSTGRES_PASSWORD: postgres
+    resource_class: medium+
```

Comment:
This is up from a default of medium?
Curious, why is this changing?

Reply:
There was a Nextflow test failing with 137

Reply:
Thinking the tests (or at least the slow ones) in this section should also be moved out to a job not called unit tests

Reply:
Yes, they've been failing intermittently on other branches too, without my changes, including [develop](https://app.circleci.com/pipelines/github/dockstore/dockstore/5763/workflows/2f485201-3ffd-4c53-bf2c-c771c1adf912/jobs/9860), so I upped it.

I'll take a quick look and see if there are obvious candidates to be split out.